source/AI.h

```diff
@@ -126,7 +126,7 @@ class AI {
 	static double TurnToward(const Ship &ship, const Point &vector, const double precision = 0.9999);
 	static bool MoveToPlanet(const Ship &ship, Command &command, double cruiseSpeed = 0.);
 	static bool MoveTo(const Ship &ship, Command &command, const Point &targetPosition,
-		const Point &targetVelocity, double radius, double slow, double cruiseSpeed = 0.);
+		const Point &targetVelocity, double radius, double slow, double cruiseSpeed = 0., const Point &direction = Point());
```

Comment:
```suggestion
		const Point &targetVelocity, double radius, double slow, double cruiseSpeed = 0., const Point &finalDirection = Point());
```
to match the .cpp version